glide.lock

```diff
@@ -152,13 +154,17 @@ imports:
   - zapcore
   - zaptest/observer
 - name: golang.org/x/net
-  version: 8a410e7b638dca158bf9e766925842f6651ff828
+  version: 9b4f9f5ad5197c79fd623a3638e70d8b26cef344
   subpackages:
   - context
+- name: golang.org/x/tools
+  version: 5ef16f43e633c268c7c28abdba1a0173a4d715b8
```

Comment:
This isn't the latest master, https://github.com/golang/tools/commit/5ef16f43e633c268c7c28abdba1a0173a4d715b8, it is before the revert was made.